samples/basic/multitarget_hello_world/README.rst

```diff
@@ -0,0 +1,69 @@
+.. zephyr:code-sample:: multitarget_hello_world
+   :name: Hello World for multiple board targets
+
+   Run a hello world sample on multiple board targets
```

Comment:
I am a bit torn as to whether that should be a basic sample, and I don't think advertising it using "Hello World" terminology is effective (think: what would people use as keywords to find this)

I would suggest to move this to application_development and, while of course the code itself can be a hello world, I think the name & description should be more descriptive of the use case (similar to a comment also made by @57300).

Something like the below:

```suggestion
   :name: Multi-target application

   Use sysbuild to build and run on multiple board targets
```

Reply:
I'm fine moving the sample. Shouldn't we have `samples/sysbuild`? Then we could have it under `samples/sysbuild/hello_world`, and call it "Multi-target hello world application".

Reply:
We already have `samples/application_development/sysbuild/with_mcuboot` so maybe `samples/application_development/sysbuild/hello_world` and add an index + ToC in `samples/application_development/sysbuild/` folder so that it's more visible? Or also fine to move sysbuild as a top-level folder but my point is the new sample should probably live next to the existing sysbuild mcuboot one.
Thanks!

Reply:
No objections from me, both options make sense.

Reply:
`samples/application_development/sysbuild/with_mcuboot` is an extremely long path, I'd suggest shortening things. `application_development` is also very generic...

something like
- `samples/sysbuild/mcuboot`
- `samples/sysbuild/hello_world`

feels better IMHO.

Reply:
ok since you are rescoping this, how about making it blink led-0 and led-1 (at different rates) as well? just to give some confidence that the cores are alive and running for boards where the integrated debugger only has a single uart available

Reply:
> ok since you are rescoping this, how about making it blink led-0 and led-1 (at different rates) as well? just to give some confidence that the cores are alive and running for boards where the integrated debugger only has a single uart available

sounds like a good sample, but shouldn't it be ab extra `samples/sysbuild/blinky`, following our basic samples?

Reply:
I'll propose a samples/sysbuild/blinky if this moves on, not before, though.